dev/plugins/hu.elte.txtuml.diagnostics/.classpath

```diff
@@ -1,6 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <classpath>
-	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
```

Comment:
Please revert this in `hu.elte.txtuml.api.model.execution`, as `HttpServer` is not needed there anymore.

Reply:
My request above still holds.